Dockerfile

```diff
@@ -1,4 +1,24 @@
-FROM golang:1.14-buster
+FROM python:3.8.5-buster as code
```

Comment:
`code` seems a little ambiguous to me, since the final image contains our code, too.  Would maybe `build` make sense?

Reply:
sure 👍